src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java

```diff
@@ -78,4 +85,26 @@ public synchronized String getCurrentCommandString() {
     public Publisher getChangePublisher() {
         return changePublisher;
     }
+
+    class Command {
+        String operation;
+        int posX;
+        int posY;
+    }
+
+    public synchronized void loadCommands(String loadedCommands) {
+        Gson gson = new Gson();
+        Type dataListType = new TypeToken<Collection<Command>>() {
+        }.getType();
+        List<Command> all = gson.fromJson(loadedCommands, dataListType);
+        List<DriverCommand> po = new ArrayList<DriverCommand>();
+        for (Command t : all) {
+            if (t.operation.equals("SetPositionCommand")) {
+                po.add(new SetPositionCommand(t.posX, t.posY));
+            } else if (t.operation.equals("OperateToCommand")) {
+                po.add(new OperateToCommand(t.posX, t.posY));
+            }
+        }
+        setCurrentCommand(po, "Loaded input");
```

Comment:
Can you add an optional command name to the input text.

Reply:
Can you write me an example?

Reply:
Something like:
{
"name":  "a point",
"commands": [
  {
    "operation": "SetPositionCommand",
    "posX": -20,
    "posY": -50
  },
  {
    "operation": "OperateToCommand",
    "posX": -20,
    "posY": -50
  }
] 
}